### PR TITLE
Per project versioning

### PR DESCRIPTION
**Pull request by arkpar**

### CODE REVIEW

File: libwebthree/WebThree.cpp [hunk 1]
```diff
@@ -31,7 +31,7 @@
 #include <libethereum/Defaults.h>
 #include <libethereum/EthereumHost.h>
 #include <libwhisper/WhisperHost.h>
-#include "BuildInfo.h"
+#include "ethereum/BuildInfo.h"
```
Comment (LefterisJP) [inline libwebthree/WebThree.cpp:34]: Won't work with split projects

**Comment by arkpar** [comment]
https://github.com/ethereum/webthree-helpers/pull/18
https://github.com/ethereum/libweb3core/pull/6
https://github.com/ethereum/libethereum/pull/19
https://github.com/ethereum/mix/pull/75
https://github.com/ethereum/solidity/pull/79
https://github.com/ethereum/webthree/pull/41
https://github.com/ethereum/alethzero/pull/63
These PRs go together

**Comment by debris** [comment]
besides split projects stuff, it looksgood

**Comment by arkpar** [comment]
Tested to work with split repos, just need other PRs merged

**Pull request merged**